tests/plot_test.py

```diff
@@ -0,0 +1,144 @@
+"""Test plot
+
+:copyright: Copyright (c) 2025 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+from pykern import pkunit
+
+
+def test_imageset_save_to_file():
+    from glob import glob
```

Comment:
generally we just import glob, which is the module, but since you already import pkio. just use pkoio.sorted_glob